threedi_modelchecker/checks/other.py

```diff
@@ -551,3 +551,13 @@ def get_invalid(self, session: Session) -> List[NamedTuple]:
 
     def description(self) -> str:
         return f"{self.column_name} will empty its storage faster than one timestep, which can cause simulation instabilities"
+
+
+class BetaFeaturesCheck(BaseCheck):
+    """Check that no beta features were used in the database"""
+
+    def get_invalid(self, session: Session) -> List[NamedTuple]:
+        return session.query(self.table).filter(self.column.isnot(None)).all()
+
+    def description(self) -> str:
+        return f"{self.column_name} is a beta feature, which is still under development; please do not use it yet."
```

Comment:
Beta tables generate an error for every column this way...  Which is probably fine since I guess this happens not that often...

Reply:
For the VegetationDrag beta tables, currently models.GlobalSetting.vegetation_drag_settings_id is added to the column checks. If, in the future, tables are added which are not referenced in the global settings, the two options would be
- define a new check which checks if any rows exist in the beta table and add that check to beta_features_check
- add the id column of that table to the list of beta columns